src/services/gitlab/GitLabService.spec.ts

```diff
@@ -85,31 +89,31 @@ describe('GitLab Service', () => {
   it('Returns issues in own interface', async () => {
     const mockIssue = gitLabIssueResponseFactory({});
 
-    gitLabNock.listIssuesResponse([mockIssue]);
+    gitLabNock.listIssuesResponse([mockIssue], { pagination: { page: 1, perPage: 1 } });
     gitLabNock.getGroupInfo();
 
-    const response = await service.listIssues('gitlab-org', 'gitlab');
-    expect(response).toMatchObject(listIssuesResponse());
+    const response = await service.listIssues('gitlab-org', 'gitlab', { pagination: { page: 1, perPage: 1 } });
+    expect(response).toMatchObject(listIssuesResponse(undefined, { page: 1, perPage: 1 }));
   });
 
   it('Returns issues for user in own interface', async () => {
     gitLabNock = new GitLabNock('homolova', 'ted_ontouml_kom');
 
     const mockIssue = gitLabIssueResponseFactory(issueOfUser);
 
-    gitLabNock.listIssuesResponse([mockIssue]);
+    gitLabNock.listIssuesResponse([mockIssue], { pagination: { page: 1, perPage: 1 } });
     gitLabNock.getUserInfo();
 
-    const response = await service.listIssues('homolova', 'ted_ontouml_kom');
-    expect(response).toMatchObject(listIssuesResponse(mockListIssuesResponseForUser));
+    const response = await service.listIssues('homolova', 'ted_ontouml_kom', { pagination: { page: 1, perPage: 1 } });
+    expect(response).toMatchObject(listIssuesResponse(mockListIssuesResponseForUser, { page: 1, perPage: 1 }));
   });
 
   it('Returns issue comments in own interface', async () => {
     gitLabNock = new GitLabNock('homolova', 'ted_ontouml_kom');
-    gitLabNock.listIssueCommentsResponse(1);
+    gitLabNock.listIssueCommentsResponse(1, { pagination: { page: 1, perPage: 1 } });
 
-    const response = await service.listIssueComments('homolova', 'ted_ontouml_kom', 1);
-    expect(response).toMatchObject(listIssueCommentsResponse());
+    const response = await service.listIssueComments('homolova', 'ted_ontouml_kom', 1, { pagination: { page: 1, perPage: 1 } });
+    expect(response).toMatchObject(listIssueCommentsResponse(undefined, { page: 1, perPage: 1 }));
```

Comment:
Why you didn't set `, { pagination: { page: 1, perPage: 1 } }` as default value of pagination?